docs/topics/Voice_Connections.md

```diff
@@ -35,11 +35,11 @@ The first step in connecting to a voice server (and in turn, a guild's voice cha
 }
 ```
 
-If our request succeeded, the gateway will respond with _two_ events—a [Voice State Update](#DOCS_TOPICS_GATEWAY/voice-state-update) event and a [Voice Server Update](#DOCS_TOPICS_GATEWAY/voice-server-update) event—meaning your library must properly wait for both events before continuing. The first will contain a new key, `session_id`, and the second will provide voice server information we can use to establish a new voice connection:
+If our request succeeded, the Gateway would respond with _two_ events—a [Voice State Update](#DOCS_TOPICS_GATEWAY/voice-state-update) event and a [Voice Server Update](#DOCS_TOPICS_GATEWAY/voice-server-update) event—meaning your library must properly wait for both events before continuing. The first will contain a new key, `session_id`, and the second will provide voice server information we can use to establish a new voice connection:
```

Comment:
"will" seems better, but you'd need to change "succeeded" to "succeeds". not sure what is actually better here